templates/patch-prometheus.yaml.j2

```
@@ -0,0 +1,6 @@
+spec:
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: {{ port }}
```

can you do this to just ensure that it's a string?

```suggestion
        prometheus.io/port: {{ str(port) }}
```